src/game/game.html

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
+  </head>
 
-<body>
+  <body>
     <div>game</div>
-</body>
-
-</html>
\ No newline at end of file
+    <!-- paragraf z klasą timer na potrzeby wyświetlenia zegara. Do późniejszej zimany
```

Comment:
Typo - `zimany` 😄 ... besides I propose doing it everywhere the same way (TODO) and in English, in shortcut:
`//@TODO: Temporary paragraph for displaying timer`

Reply:
again 🤦